src/services/oauth2/index.js

```diff
@@ -179,6 +182,11 @@ export default function(options){
     // Get our initialized service
     const service = app.service(options.endPoint);
 
+    // prevent regular service events from being dispatched
```

Comment:
I had to think about this for a second but I think you are right. It **should** go in there because then someone can override setup if they really need that functionality for whatever reason. We might come back to this when we address scaling services.